invoke/runners.py

```diff
@@ -216,7 +216,7 @@ def run(self, command, **kwargs):
             err_stream = sys.stderr
         # Echo
         if opts['echo']:
-            print("\033[1;37m{0}\033[0m".format(command))
+            print(u"\033[1;37m{0}\033[0m".format(command))
```

Comment:
Using UTF-8 in the Windows console is pretty buggy. You probably shouldn't do that.

There are better ways of getting full Unicode in the Windows console. Python doesn't use them by default (because doing so requires violating the Unix/C assumption that the console is just like any other file stream, and it's hard to do that in a compatible way). The 3rd party module ```win-unicode-console``` implements them in a nearly-transparent way, if you're interested.